lib/commands/help.js

```diff
@@ -95,13 +95,15 @@ class Help extends BaseCommand {
       args = ['emacsclient', ['-e', `(woman-find-file '${man}')`]]
     }
 
-    return spawn(...args, { stdio: 'inherit' }).catch(err => {
```

Comment:
npm was not using the return data from promise-spawn here so this is ok

https://github.com/npm/promise-spawn/blob/0aaf6f0f37e5b468277c8a4d8200b5ab18a4b387/lib/index.js#L27-L33